src/db.rs

```diff
@@ -332,6 +332,7 @@ CREATE table review_prefs (
     assigned_prs INT[] NOT NULL DEFAULT array[]::INT[]
 );",
     "
+CREATE EXTENSION intarray;
```

Comment:
I enabled this Postgres extension to be able to use `sort()` and `uniq()` when working on the arrays. We need to keep these INSERTs idempotent (see comment in `pr_tracking.rs`)